Source/ChilliSource/Rendering/Base/RenderCommandCompiler.cpp

```diff
@@ -333,11 +333,11 @@ namespace ChilliSource
     }
     
     //------------------------------------------------------------------------------
-    RenderCommandBufferUPtr RenderCommandCompiler::CompileRenderCommands(const TaskContext& taskContext, const std::vector<TargetRenderPassGroup>& targetRenderPassGroups, RenderCommandListUPtr preRenderCommandList,
-                                                                          RenderCommandListUPtr postRenderCommandList, RenderFrameData renderFrameData) noexcept
+    RenderCommandBufferUPtr RenderCommandCompiler::CompileRenderCommands(const TaskContext& taskContext, IAllocator* frameAllocator, const std::vector<TargetRenderPassGroup>& targetRenderPassGroups, RenderCommandListUPtr preRenderCommandList,
```

Comment:
again, why pass by rvalue ref? This is dangerous as the caller doesn't know they've moved the data. an explicit std::move() achieves the same outcome with less risk.